libraries/libraries.sls

```diff
@@ -0,0 +1,12 @@
+include:
```

Comment:
I created this as a place to install various libraries. Let me know if you think each library should get its own `.sls` file.

Reply:
That sounds pretty reasonable.  Until something requires a bunch of moving files around simple is good.